springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/types/AsyncAPI.java

```diff
@@ -71,25 +74,30 @@ public class AsyncAPI {
     private Map<String, Server> servers;
 
     /**
-     * <b>Required.</b>
-     * The available channels and messages for the API.
-     * Channels are also known as "topics", "routing keys", "event types" or "paths".
+     * The channels used by this
+     * <a href="https://www.asyncapi.com/docs/reference/specification/v3.0.0#definitionsApplication">application</a>.
+     * </p>
+     * An identifier for the described channel. The channelId value is case-sensitive. Tools and libraries MAY
+     * use the channelId to uniquely identify a channel, therefore, it is RECOMMENDED to follow common programming
+     * naming conventions.
      */
     @NonNull
-    private Map<String, ChannelItem> channels;
+    private Map<String, ChannelObject> channels;
+
+    /**
+     * Holds a dictionary with all the operations this application MUST implement.
+     * </p>
+     * If you're looking for a place to define operations that MAY or MAY NOT be implemented by the
+     * <a href="https://www.asyncapi.com/docs/reference/specification/v3.0.0#definitionsApplication">application</a>
+     * consider defining them in components/operations.
+     */
+    @JsonProperty(value = "operations")
+    private Map<String, Operation> operations;
 
     /**
      * Holds a set of reusable objects for different aspects of the AsyncAPI specification.
      * All objects defined within the components object will have no effect on the API unless they are explicitly
      * referenced from properties outside the components object.
      */
     private Components components;
```

Comment:
I didn't manage to remove this AsyncAPI class and replace it by the one from the `springwolf-asyncapi` module. The reason is that this `components` field is used in different places. 

Maybe would be worth it to drop a comment here if this class is intended to stay _"duplicated"_